common/templates/profile.modal.html

```diff
@@ -20,15 +20,15 @@ <h2 class="modal-title text-center">{{ctrl.user}}</h2>
                     </ul>
                 </td>
             </tr>
-            <tr ng-if="ctrl.groupList">
-                <td rowspan="{{ctrl.groupList.length}}" class="text-left">Groups</td>
+            <tr ng-if="ctrl.globusGroupList">
+                <td rowspan="{{ctrl.globusGroupList.length}}" class="text-left">Groups</td>
```

Comment:
What is this rowspan used for? do we need it? And if we do, should this include the `otherGroups` as well?

Reply:
I don't think we need to define rowspan here. It has no effect.